Extend the documentation a bit (#70)
Foxboron/sbctl: docs/sbctl.8.txt

                Sign all the generated bundles.

**remove-bundle** <NAME>::
        Removes a bundle from the list. This does not delete the bundle itself.

**list-bundles**::
        List all registered bundles to generate.

Bundles
-------

> Bundles are EFI binaries which pack all the resources (initramfs, kernel, etc)

Reviewer: `EFI executables` maybe? A little bit unsure if "binaries" convey the same meaning? I also think we should write `(initramfs, kernel and cmdline)`.

into a single file which can be easily signed. They also don't read parameters

> from external files, which makes them more tamper-proof.

Reviewer: Add a line about the security guarantees? `Normally only the kernel is signed with your secure boot keys, this leaves the kernel cmdline and initramfs unprotected.`

> When a bundle is generated, its metadata is stored into the bundle database

Author: Not sure if I prefer `metadata` or `manifest` here. Thoughts?

Reviewer: `configuration` is better I think.

(See **FILES**). Subsequent executions of *sbctl generate-bundles* will rebuild
these bundles, so you don't need to re-specify all parameters after each system
update.

Hint: systemd-boot will automatically show entries for any bundles found in

> *$EFI/Linux/+++*+++.efi*.

Reviewer: The proper way to write the path is `esp/EFI/Linux/+++*+++.efi` as we do on the wiki.

                Sign all the generated bundles.

**remove-bundle** <NAME>::
        Removes a bundle from the list. This does not delete the bundle itself.

**list-bundles**::
        List all registered bundles to generate.

Bundles
-------

> Normally, only the kernel is signed with your secure boot keys. This leaves the
> cmdline and initramfs unprotected.

Reviewer 2: ```suggestion
Normally, only the kernel is signed with your secure boot keys. This means the
kernel command line and initramfs can be changed without possibility of verification.
```

or something like that? Mainly, I'd like to have `kernel command line` spelled out.

Bundles are EFI executables which pack all three (initramfs, kernel and

> cmdline) into a single file which can easily be signed. Avoiding any unsigned

Reviewer 2: ```suggestion
cmdline) into a single file which can be easily signed. Avoiding any unsigned
```

files during boot makes the whole process more tamper-proof.

When a bundle is generated, its configuration is stored into the bundle

> database (See **FILES**). Subsequent executions of *sbctl generate-bundles*

Reviewer 2: ```suggestion
database (see **FILES**). Subsequent executions of *sbctl generate-bundles*
```

Bundles
-------

> Normally, only the kernel is signed with your secre boot keys. This leaves the

Reviewer: "secure"

cmdline and initramfs unprotected.

Bundles are EFI executables which pack all three (initramfs, kernel and
cmdline) into a single file which can easily be signed. Avoiding any unsigned
files during boot makes the whole process more tamper-proof.

When a bundle is generated, its configuration is stored into the bundle
database (See **FILES**). Subsequent executions of *sbctl generate-bundles*
will rebuild these bundles, so you don't need to re-specify all parameters
after each system update.

Hint: systemd-boot will automatically show entries for any bundles found in

> *esp/Linux/+++*+++.efi*.

Reviewer: Still missing the `EFI` top-level directory :)
